src/data-workspace/data-entry-cell/entry-field-input.js

```diff
@@ -91,8 +91,7 @@ export function EntryFieldInput({
 
     const onFocus = useCallback(() => {
         setHighlightedFieldId({ de, coc })
-        setRightHandPanel('')
-    }, [de, coc, setHighlightedFieldId, setRightHandPanel])
+    }, [de, coc, setHighlightedFieldId])
```

Comment:
@Mohammer5 I removed `setRightHandPanel('')` since it caused the right panel to close whenever the user focuses on a cell. This was added as part of the performance enhancements but it's not the expected behaviour, right?

Reply:
Yes, this will be removed anyway in #73